packages/sanity/playwright-ct/tests/utils/testHelpers.tsx

```diff
@@ -12,9 +12,7 @@ export function testHelpers({page}: {page: PlaywrightTestArgs['page']}) {
       await $overlay.focus()
       await page.keyboard.press('Space')
     }
-    await $pteField
-      .locator(`[data-testid='pt-editor__toolbar-card']`)
-      .waitFor({state: 'visible', timeout: 1000})
+    await $overlay.waitFor({state: 'detached', timeout: 1000})
```

Comment:
Minor change – note that we now check for the overlay to be detached as opposed to checking for toolbar visibility, since this can fail with PTE inputs with `hideToolbar=true`